packages/ckeditor5-engine/docs/_snippets/framework/extending-content-custom-element-converter.js

```diff
@@ -123,19 +128,51 @@ function insertViewElements( data, conversionApi, infoBox, infoBoxTitle, infoBox
 		infoBoxContent
 	);
 
-	// The default mapping between the model <infoBox> and its view representation.
+	// The mapping between the model <infoBox> and its view representation.
 	conversionApi.mapper.bindElements( data.item, infoBox );
-	// However, since the model <infoBox> content needs to end up in the inner
-	// <div class="info-box-content">, you need to bind one with another overriding
-	// a part of the default binding.
-	conversionApi.mapper.bindElements( data.item, infoBoxContent );
 
 	conversionApi.writer.insert(
 		conversionApi.mapper.toViewPosition( data.range.start ),
 		infoBox
 	);
 }
 
+function createModelToViewPositionMapper( view ) {
```

Comment:
The code here should be an exact copy of the code in the guide itself (comments are missing) for easier comparison and maintenance.